genai-perf/docs/tutorial.md

```diff
@@ -180,17 +180,17 @@ Output token throughput (per sec): 290.24
 Request throughput (per sec): 2.57
 ```
 
-## Profile GPT2 running on OpenAI Chat API-Compatible Server <a id="openai-chat"></a>
+## Profile Zephyr running on OpenAI Chat API-Compatible Server <a id="openai-chat"></a>
 
-### Run GPT2 on [OpenAI Chat Completions API](https://platform.openai.com/docs/api-reference/chat)-compatible server
+### Run Zephyr on [OpenAI Chat Completions API](https://platform.openai.com/docs/api-reference/chat)-compatible server
 
 <details>
 <summary>See instructions</summary>
 
 Run the vLLM inference server:
 
 ```bash
-docker run -it --net=host --gpus=all vllm/vllm-openai:latest --model gpt2 --dtype float16 --max-model-len 1024
+docker run -it --net=host --gpus=all vllm/vllm-openai:latest --model HuggingFaceH4/zephyr-7b-beta --dtype float16 --max-model-len 1024
```

Comment:
the context window of zephyr is much longer than 1024 (e.g. 32,768) so we may want to just remove the `--max-model-len 1024` part
```suggestion
docker run -it --net=host --gpus=all vllm/vllm-openai:latest --model HuggingFaceH4/zephyr-7b-beta --dtype float16
```

Reply:
Great catch. Thanks for reviewing so quickly and well, Hyunjae!